src/server.h

```diff
@@ -2773,6 +2778,10 @@ typedef struct {
     int minex, maxex; /* are min or max exclusive? */
 } zlexrangespec;
 
+/* flags for resetErrorCountSnapshot */
```

Comment:
if you change the function name, don't forget to update it here.

Reply:
Fixed